src/index.d.ts

```diff
@@ -389,7 +389,12 @@ export interface StopProps {
 export const Stop: React.ComponentClass<StopProps>;
 export type Stop = React.ComponentClass<StopProps>;
 
-export interface SvgProps extends GProps, ReactNative.ViewProperties {
+export type AdditionalProps = {
```

Comment:
I think we should not add it to `SvgProps` since `Svg` does not have `onError` prop. Instead, I think it should be added to `UriProps` and `XmlProps`, similar to how it's done here: https://github.com/react-native-svg/react-native-svg/blob/2fa9645cf6588231b8435301f736a721d8c0a311/src/xml.tsx#L89. So the code updated in `index.d.ts` should be `UriProps`: https://github.com/react-native-svg/react-native-svg/blob/2fa9645cf6588231b8435301f736a721d8c0a311/src/index.d.ts#L545 and the one below: `XmlProps`.